org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/ADTabpanel.java

```diff
@@ -2123,4 +2123,16 @@ else if (getGridTab() != null)
 		
 		return hasQuickForm;
 	}
+
+	@Override
+	public boolean isEnableCustomizeButton()
+	{
+		return isGridView();
+	}
+
+	@Override
+	public boolean isEnableProcessButton()
+	{
+		return getToolbarButtons().size() > 0 && !gridTab.isNew();
```

Comment:
this is a good candidate to move to interface as default method.

Reply:
@hengsin I did not understand this suggestion. isEnableProcessButton, isEnableCustomizeButton are already in IADTabpanel interface.

Reply:
Since it is unlikely for implementation class to change the "return getToolbarButtons().size() > 0 && !gridTab.isNew();" logic, instead of making it a must for every implementation of IADTabpanel interface to implement the method, change it to public default boolean isEnableProcessButton() {return getToolbarButtons().size() > 0 && !getGridTab().isNew();} at IADTabpanel interface.

Reply:
In latest commit, I made this change. Please verify that I have done correctly.